packages/client-config/src/client-config-writer/client_config_to_mobile_legacy_converter.ts

```diff
@@ -74,6 +74,17 @@ export class ClientConfigMobileConverter {
           },
         },
       };
+      let oauth;
+      if (clientConfig.oauth) {
+        oauth = {
+          WebDomain: clientConfig.oauth.domain,
+          Scopes: clientConfig.oauth.scope,
+          SignInRedirectURI: clientConfig.oauth.redirectSignIn,
+          SignOutRedirectURI: clientConfig.oauth.redirectSignOut,
+          AppClientId: clientConfig.oauth.clientId,
+        };
+        authConfig.plugins.awsCognitoAuthPlugin.Auth.Default.OAuth = oauth;
+      }
```

Comment:
Why do we need `oauth`? How about the following?
```suggestion
      if (clientConfig.oauth) {
        authConfig.plugins.awsCognitoAuthPlugin.Auth.Default.OAuth = {
          WebDomain: clientConfig.oauth.domain,
          Scopes: clientConfig.oauth.scope,
          SignInRedirectURI: clientConfig.oauth.redirectSignIn,
          SignOutRedirectURI: clientConfig.oauth.redirectSignOut,
          AppClientId: clientConfig.oauth.clientId,
        };
      }
```